src/components/Themes/AppTheme01/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './AppTheme01'
```

Comment:
Rename?

Reply:
it has been removed and will be reflect with the next push